### PR TITLE
About us

### PR DESCRIPTION
**Pull request by ThomasDruart**
- 2 members added
- text updated
- npx prettier done

### CODE REVIEW

**Review by AmjadLille — DISMISSED** [review]

File: src/pages/about/index.jsx [hunk 1]
```diff
@@ -2,34 +2,41 @@ import React from "react";
...
 import portraitItems from "./Data";
 import { AboutContainer, Cards, Pledge, PledgeBox, Partners } from "./style";
+import Title from "../../components/Title/index";
 
 const About = () => {
   return (
     <AboutContainer>
       <Pledge>
-        <h1>Notre Engagement</h1>
+        <Title titleName={"Notre Engagements"} />
```
Comment (LoicBrassart) [inline src/pages/about/index.jsx]: "Notre Engagement" or "Nos Engagements"

File: src/pages/about/index.jsx [hunk 2]
```diff
@@ -2,34 +2,41 @@ import React from "react";
...
             </p>
           </div>
           <img src="/images/atelierbis.jpg" alt="atelier" />
         </PledgeBox>
       </Pledge>
 
-      <div>
-        <h1>L'Equipe de Toi Demain</h1>
+      <div id="equipe">
+        <Title titleName={"L'Equipe de Toi Demain"} />
```
Comment (LoicBrassart) [inline src/pages/about/index.jsx]: Curly braces are not needed on strings: `<Title titleName="L'Equipe de Toi Demain" />`

File: src/pages/about/index.jsx [hunk 3]
```diff
@@ -39,7 +46,7 @@ const About = () => {
       </div>
 
       <div>
-        <h1>Nos partenaires</h1>
+        <Title titleName={"Nos partenaires"} />
```
Comment (LoicBrassart) [inline src/pages/about/index.jsx]: Curly braces are not needed on strings: <Title titleName="Nos partenaires" />

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Minor Evils

**Review by AmjadLille — APPROVED** [review]

**Review by LoicBrassart — APPROVED** [review]
NmE

**Pull request merged**